tools/misc/browse_dataset.py

```diff
@@ -189,6 +190,10 @@ def main():
             cfg.data.train, default_args=dict(filter_empty_gt=False))
     except TypeError:  # seg dataset doesn't have `filter_empty_gt` key
         dataset = build_dataset(cfg.data.train)
+
+    # use only first dataset for ConcatDataset
+    if isinstance(dataset, ConcatDataset):
+        dataset = dataset.datasets[0]
```

Comment:
Maybe we can move this judgement to between line 64 and line 65 (the line id before modification)? Like:

```
if cfg.data.train['type'] == 'ConcatDataset':
        cfg.data.train = cfg.data.train.datasets[0]
```